src/main/java/com/dev/cinema/exceptions/AuthenticationException.java

```diff
@@ -1,11 +1,18 @@
 package com.dev.cinema.exceptions;
 
+import org.apache.log4j.Logger;
+
 public class AuthenticationException extends Exception {
+
+    private static final Logger logger = Logger.getLogger(RuntimeException.class);
+
     public AuthenticationException(String message, Throwable cause) {
         super(message, cause);
+        logger.error(message,cause);
```

Comment:
this is redundant, let's remove logging from this class
exceptions are logged automatically